_IBC/cosmoshub-kopi.json

```diff
@@ -2,22 +2,22 @@
     "$schema": "../ibc_data.schema.json",
     "chain_1": {
         "chain_name": "cosmoshub",
-        "client_id": "07-tendermint-1332",
-        "connection_id": "connection-1065"
+        "client_id": "07-tendermint-1382",
+        "connection_id": "connection-1105"
     },
     "chain_2": {
         "chain_name": "kopi",
-        "client_id": "07-tendermint-4",
-        "connection_id": "connection-8"
+        "client_id": "07-tendermint-19",
+        "connection_id": "connection-43"
     },
     "channels": [
         {
             "chain_1": {
-                "channel_id": "channel-1253",
+                "channel_id": "channel-1351",
```

Comment:
The old channel doesn't appear to be used much, but is there evidence that there is no abandoned supply of ATOM on Kopi?

![image](https://github.com/user-attachments/assets/ff39d472-2d14-4d2d-a0a8-e6b42a557a2a)